ballerina/tests/07-local-transaction-test.bal

```diff
@@ -40,33 +40,36 @@ public class SQLDefaultRetryManager {
 
 @test:BeforeGroups { value:["local-transaction"] }
 isolated function beforeTransactionFunc() returns sql:Error? {
-    Client oracledbClient = check new(HOST, USER, PASSWORD, DATABASE, PORT);
-    sql:ExecutionResult result = check dropTableIfExists("LocalTranCustomers");
-    result = check oracledbClient->execute("CREATE TABLE LocalTranCustomers(" +
-        "id NUMBER GENERATED ALWAYS AS IDENTITY, " +
-        "firstName VARCHAR2(100), " +
-        "lastName VARCHAR2(100), " +
-        "registrationID NUMBER, " +
-        "creditLimit VARCHAR2(100), " +
-        "country VARCHAR2(100), " +
-        "PRIMARY KEY (id) " +
-        ")"
+    sql:ConnectionPool pool = {maxOpenConnections: 3, minIdleConnections: 1};
+    Client oracledbClient = check new(HOST, USER, PASSWORD, DATABASE, PORT, connectionPool = pool);
+    sql:ExecutionResult result = check dropTableIfExists("LocalTranCustomers", oracledbClient);
+    result = check executeQuery(`CREATE TABLE LocalTranCustomers(
+        id NUMBER GENERATED ALWAYS AS IDENTITY,
+        firstName VARCHAR2(100),
+        lastName VARCHAR2(100),
+        registrationID NUMBER,
+        creditLimit VARCHAR2(100),
+        country VARCHAR2(100),
+        PRIMARY KEY (id)
+        )`
```

Comment:
Shall we prioritize moving table creation and type creation to SQL scripts and run when setting up the database. issue ballerina-platform/ballerina-standard-library#1729 after your current tasks. it will also reduce the total connections used.